tests/apollo/test_skvbc_backup_restore.py

```diff
@@ -295,7 +297,7 @@ async def _stop_random_replicas_with_delay(bft_network, delay=10, exclude_replic
         all_replicas = bft_network.all_replicas(without=exclude_replicas)
         random.shuffle(all_replicas)
         for replica in all_replicas:
-            print("stopping replica: ", replica)
+            SkvbcBackupRestoreTest.logger.info(f"stopping replica: {replica}")
```

Comment:
why not simply say: `logger.info(f"stopping replica: {replica}")` ?

Same everywhere else - why would we need to prepend the class name?